Content.Shared/Fluids/Components/DrainComponent.cs

```diff
@@ -33,6 +33,12 @@ public sealed partial class DrainComponent : Component
     [DataField("autoDrain"), ViewVariables(VVAccess.ReadOnly)]
     public bool AutoDrain = true;
 
+    /// <summary>
+    /// Does examining this drain show available volume?
+    /// </summary>
+    [DataField("examineAvailableVolume"), ViewVariables(VVAccess.ReadOnly)]
```

Comment:
```suggestion
    [DataField, ViewVariables(VVAccess.ReadOnly)]
```
Not sure why this needs to be read only either.

Reply:
this shouldn't exist, we have an ExaminableSolutionComponent